commands/sync.go

```diff
@@ -58,21 +79,38 @@ var SyncStatusCmd = &cli.Command{
 			} else {
 				heightDiff = 0
 			}
-			fmt.Printf("\tBase:\t%s\n", base)
-			fmt.Printf("\tTarget:\t%s (%d)\n", target, theight)
-			fmt.Printf("\tHeight diff:\t%d\n", heightDiff)
-			fmt.Printf("\tStage: %s\n", ss.Stage)
-			fmt.Printf("\tHeight: %d\n", ss.Height)
-			if ss.End.IsZero() {
-				if !ss.Start.IsZero() {
-					fmt.Printf("\tElapsed: %s\n", time.Since(ss.Start))
+
+			switch output {
+			case "json":
+				j, err := json.Marshal(SyncStatus{Stage: maxStateSync, Height: max})
+				if err != nil {
+					return err
 				}
-			} else {
-				fmt.Printf("\tElapsed: %s\n", ss.End.Sub(ss.Start))
+				fmt.Printf(string(j) + "\n")
+			case "":
```

Comment:
could call this "pretty" or "verbose" and set the default value of the `--output` flag as such, but not a big deal